amfitrack:
  ros__parameters:
    relativePos: 'human_robot_pos'
    tf_frame_id: 'base_link'
    base_link: 'base_link'
    tf_child_name: 'human_link'
    delta_t: 0.05
    num_step: 10

